_posts/2021-11-12-github-copilot.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Github Copilot - let AI write your code?
+author: arto
+excerpt: >
+  Using tools to increase effectiveness has been a long-standing trend in the software development community. Latest wave comes in form of GPT-3, OpenAI, and ultimately Github Copilot. Let's try to answer the question: Can AI write your code?
```

Comment:
Can AI write your code? -> Can AI-generated code be better than yours? :trollface:

Reply:
Appreciated, but I deliberately like the original trolling version :)